server/routes/api/section.js

```diff
@@ -159,4 +165,45 @@ router.get('/updateSection', (req, res) => {
     });
 });
 
-module.exports = router;
\ No newline at end of file
+module.exports = router;
+
+// ------------- Combined Seating Layout and Course Methods -------------
+
+// gets information regarding a course given a student/prof id
+router.get('/getCourseView', (req, res) => {
+    // this is the professor ID
+    let courseID = req.body.courseID;
+
+    Course.findOne( {_id: courseID} , function(err, course){
+        if(err){
+            console.log(err);
+            return res.status(500).send();
+        }
+
+        seatingLayout = SeatingLayout.findOne({_id: course.seating_layout}, function(err, seatingLayout){
```

Comment:
Might want to remove the assignment since we're not using it (it also confused me a little since the same variable name is used in the callback function)
```suggestion
        SeatingLayout.findOne({_id: course.seating_layout}, function(err, seatingLayout){
```

Reply:
Yeah that should be removed. I was trying some other stuff and forgot to remove the assignment.